### PR TITLE
Remove tiered identity

### PR DESCRIPTION
**Pull request by apc999**
### What changes are proposed in this pull request?

Remove tiered identity

### Why are the changes needed?

This feature is no longer supported

### Does this PR introduce any user facing changes?

N/A
Entry points have been removed previously. This PR removes the src code

### CODE REVIEW

File: integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseRestServiceHandler.java [hunk 1]
```diff
@@ -1,59 +0,0 @@
...
- * (the "License"). You may not use this work except in compliance with the License, which is
- * available at www.apache.org/licenses/LICENSE-2.0
- *
- * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
- * either express or implied, as more fully set forth in the License.
- *
- * See the NOTICE file distributed with this work for information regarding copyright ownership.
- */
-
-package alluxio.fuse;
```
Comment (jiacheliu3) [inline integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseRestServiceHandler.java:12]: this one is technically irrelevant to this change?

**Review by jiacheliu3 — APPROVED** [review]
LGTM except one comment on an irrelevant change. I'll leave it up to your judgement :)
Thanks!

File: integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseRestServiceHandler.java (hunk 1 above)
Comment (apc999) [reply integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseRestServiceHandler.java]: reverted

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Pull request merged**